tests/unit/test_expression_tree/test_functions.py

```diff
@@ -127,6 +129,21 @@ def test_function_unnamed(self):
             fun.name, "function (<class 'scipy.interpolate.interpolate.interp1d'>)"
         )
 
+    def test_to_equation(self):
+        a = pybamm.Symbol("a", domain="test")
+
+        # Test _sympy_operator
+        with self.assertRaises(NotImplementedError):
+            pybamm.Erf(a).to_equation()
```

Comment:
Is there a symbol that we know for sure will never be implemented? Because `Erf` should be fairly easy to bring in.